owl/closure.scm

```diff
@@ -26,7 +26,7 @@
 
       (define (small-value? val)
          (or
-            (and (fixnum? val) (>= val -127) (< val 127))
```

Comment:
Comparisons like this typically date back to when only fixed arity functions were present.